xmpp.go

```diff
@@ -818,6 +818,14 @@ func (c *Client) Recv() (stanza interface{}, err error) {
 						return handleAvatarMetadata(p.Items[0].Body,
 							v
 					}*/
+				default:
+                                res, err := xml.Marshal(v.Query)
```

Comment:
Please do `go fmt ./...`

Reply:
I did a `gofmt -w` before but it also changed code that I didn't touch so I reverted it. Hope the part I changed is no properly indented.

Reply:
Thank you